src/lib/inclusion_user_callbacks.ts

```diff
@@ -0,0 +1,88 @@
+import { InclusionGrant, InclusionUserCallbacks } from "zwave-js";
+import { createDeferredPromise } from "alcalzone-shared/deferred-promise";
+import { Client, ClientsController } from "./server";
+
+export const inclusionUserCallbacks = (
+  clientsController: ClientsController,
+  client?: Client
+): InclusionUserCallbacks => {
+  return {
+    grantSecurityClasses: (
+      requested: InclusionGrant
+    ): Promise<InclusionGrant | false> => {
```

Comment:
Since you specify the return type of the surrounding method as `InclusionUserCallbacks`, the function types should entirely be inferred by TypeScript.
If anything, manually typing them again could introduce conflicts.

```suggestion
    grantSecurityClasses: (requested) => {
```